### PR TITLE
get_api_endpoint_from_stage: teach this about the antivirus api being on port 5008

### PR DESCRIPTION
**Pull request by risicle**

### CODE REVIEW

File: dmutils/env_helpers.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def get_api_endpoint_from_stage(stage, app='api'):
     dev_ports = {
         'api': 5000,
         'search-api': 5001,
+        'antivirus-api': 5008,
```
Comment (katstevens) [inline dmutils/env_helpers.py]: I think we have tests for these as well - `TestGetAPIEndpointFromStage`

Comment (risicle) [reply dmutils/env_helpers.py]: Ohhhhright

Comment (risicle) [reply dmutils/env_helpers.py]: Fixed

**Comment by risicle** [comment]
:thunderbird_2:

**Pull request merged**